python/src/opendp/measurements.py

```diff
@@ -282,10 +296,15 @@ def make_base_discrete_laplace(
 def then_base_discrete_laplace(
     scale,
     QO: Optional[RuntimeTypeDescriptor] = None
-):
-    r"""Partial constructructor which uses the preceding output domain and metric as its input domain and metric. See also:
+):  
+    r"""partial constructor of make_base_discrete_laplace
 
-    :py:func:`opendp.measurements.make_base_discrete_laplace`
+    .. seealso:: 
+      Delays application of `input_domain` and `input_metric` in :py:func:`opendp.measurements.make_base_discrete_laplace`
+
+    :param scale: Noise scale parameter for the laplace distribution. `scale` == standard_deviation / sqrt(2).
+    :param QO: Data type of the output distance and scale. `f32` or `f64`.
```

Comment:
Offer a fully qualified name for `f32`?
```suggestion
    :param QO: Optional data type of the output distance and scale. `f32` or `f64`.
```